nss-cohort-37/Bangazon-Site-Green-Monkeys
# added initial migration

# Description

Added the Initial migration (accidental typo * spelled 'initil' lol) 



## Type of change




# Testing Instructions

- Open your package manager console and run `Update-Database`
- The build should be successful 
- Refresh your SQL server object explorer and verify that `BangazonSite` database was added



# Checklist:
